### ISSUE TITLE
InPlaceABNSync forward result is inconsistent with nn.BatchNorm2d + nn.LeakyReLU

### ISSUE THREAD

**Issue by geekboood** [mapillary/inplace_abn#126]
Hello! I found that InPlaceABNSync forward result is inconsistent with nn.BatchNorm2d + nn.LeakyReLU. Is there anyway we can train the model with InPlaceABNSync and inference it with PyTorch native implementations?

**Comment by rotabulo** [comment]
@geekboood please refer to https://github.com/mapillary/inplace_abn/issues/92
